src/test/java/redis/clients/jedis/tests/commands/ConnectionHandlingCommandsTest.java

```diff
@@ -5,20 +5,19 @@
 import org.junit.Test;
 
 import redis.clients.jedis.BinaryJedis;
-import redis.clients.jedis.HostAndPort;
-import redis.clients.jedis.tests.HostAndPortUtil;
+import redis.clients.jedis.Jedis;
 
-public class ConnectionHandlingCommandsTest extends JedisCommandTestBase {
-  protected static HostAndPort hnp = HostAndPortUtil.getRedisServers().get(0);
```

Comment:
While taking only host is not valid, I think we still need to rely on this. Could we provide host and port for Jedis and BinaryJedis instance from this?

Reply:
addressed